README.md

```diff
@@ -169,8 +173,17 @@ OnePlus | Nord N200 | [dre](https://wiki.lineageos.org/devices/dre) | | tested
 OnePlus | 9 | lemonade | | under development
 </details>
 
-And more to come!
+<details><summary><b>Xiaomi</b></summary>
+
+Vendor | Device Name                      | CodeName                                               | Models                                   | Status
+---|----------------------------------|--------------------------------------------------------|------------------------------------------|---
+Xiaomi | Redmi Note 7                     | [lavender](https://wiki.lineageos.org/devices/lavender) | lavender                                 | tested
+Xiaomi | Redmi Note 8 / 8T                | [ginkgo](https://wiki.lineageos.org/devices/ginkgo)    | ginkgo / willow                          | untested
+Xiaomi | Redmi Note 10S / 11SE / Poco M5S | [rosemary](https://wiki.lineageos.org/devices/rosemary) | rosemary / maltose / secret / rosemary_p | untested
+Xiaomi | Redmi 9A / 9C / 9AT / 9i / 9A Sport / 10A / 10A Sport | [garden](https://wiki.lineageos.org/devices/garden)    | garden / dandelion / blossom / angelican          | tested
```

Comment:
This url does not exist

Reply:
Ups... Copy - paste ! `garden` was never officially supported by Lineage. Which URL should I replace it with?

Reply:
I would also prefer, to have one config file per device code (if they are not very similar) and treat them as separate devices.

Reply:
`dandelion` and `angelican` are official Xiaomi namecodes.
`garden` and `blossom` are names of unified groups of codenames, given by ROM developers.
A `garden` ROM will fit for `dandelion` device, for example. So as far I understand, we cannot divide config files per device code, isn't it ?

Reply:
> 

What do you mean by that? - Do you mean having an extra device code for 2 configs or multiple device codes for one config? (Sorry, I am just a bit confused)

Reply:
I may forgot some (it's a mess), but globally : 

Official Xiaomi codenames :
- Redmi 9C : angelica
- Redmi 9C NFC : angelican
- Redmi 9A/10A/10A Sport/9i/9A Sport : dandelion
- Poco C3/C31 : angelicain

Community names :
- Redmi 9A/9C/9i/9 India : garden
- All of the above + Redmi 9 Active/9AT/9I Sport : blossom

Conclusion : I forgot some codenames...

Reply:
Haha, thats confusing :grinning: 

Then it might be a good idea, to start displaying multiple device codes (or the actually found one) as well as multiple possible names for devices (or based on the code) to the user. I could see this causing confusion for users (we recently also saw this for Samsung devices :D